packages/@adobe/spectrum-css-temp/components/slider/index.css

```diff
@@ -58,9 +58,35 @@ governing permissions and limitations under the License.
   touch-action: none;
 
   width: var(--spectrum-global-dimension-size-2400);
+  min-inline-size: var(--spectrum-slider-min-width);
```

Comment:
I thought textfield had the min width on the input parent not the field and input parent parent.

Reply:
Yeah honestly I don't know which is right. I am going to bring it up with Spectrum because I don't think I was specific enough when I asked them originally.